docs/serverless/03-02-function-specification.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Function's specification
+type: Details
+---
+
+Serverless in Kyma allows you to create Functions in both Node.js (10 & 12) and Python (3.8). Although the Function's interface is unified, its specification differs depending on the runtime used to run the Function.
+
+## Signature
+
+Function's code is represented by a handler that is a method processing events. When your Function is invoked, it runs this handler method to request event details and return a given response.
+
+All Functions have a predefined signature with elements common for all available runtimes:
+- Functions' code must be introduced by the `main` handler name.
+- Functions must accept two arguments that are passed to the Function handler:
+    - `event`
+    - `context`
+
+See these signatures for each runtime:
+
+<div tabs name="signature" group="function-specification">
+  <details>
+  <summary label="Node.js">
+  Node.js
+  </summary>
+
+```bash
+module.exports = {
+  main: function (event, context) {
+    return
+  }
+}
+```
+
+</details>
+<details>
+<summary label="Python">
+Python
+</summary>
+
+```bash
+def main(event, context):
+    return
+```
+
+</details>
+</div>
+
+### Event object
+
+The `event` object contains information about the event the Function refers to. For example, an API request event holds the HTTP request object.
+
+Functions in Kyma accept [CloudEvents](https://cloudevents.io/) (**ce**) with the following specification:
+
+```json
+...
+{
+    "ce-type": "com.github.pull_request.opened",
+    "ce-source": "/cloudevents/spec/pull/123",
+    "ce-eventtypeversion": "v1",
+    "ce-specversion": "1.0",
+    "ce-id": "abc123",
+    "ce-time": "2020-12-20T13:37:33.647Z",
+    "data": {BUFFER},
+    "extensions": {
+        "request": {INCOMING_MESSAGE},
+        "response": {SERVER_RESPONSE},
+    }
+}
+```
+
+See the detailed descriptions of these fields:
+
+| Field | Description |
+|-------|-------------|
+| **ce-type** | Type of the CloudEvent data related to the originating occurrence |
+| **ce-source** | Unique context in which an event happened and can relate to an organization or a process |
+| **ce-eventtypeversion** | The version of the CloudEvent type |
+| **ce-specversion** | Version of the CloudEvent specification used for this event |
+| **ce-id** | Unique identifier of the event |
+| **ce-time** | Time at which the event was sent |
+| **data** | [Buffer](https://nodejs.org/api/buffer.html) that contains the request body. It can be either JSON or a string, depending on a request type |
```

Comment:
```suggestion
| **data** | [Buffer](https://nodejs.org/api/buffer.html) that contains the request body. It can be either JSON or a string, depending on the request type. |
```